cime_config/machines/config_batch.xml

```diff
@@ -304,13 +304,44 @@
     </directives>
   </batch_system>
 
+  <batch_system type="slurm_single_node" >
+    <batch_query per_job_arg="-j">squeue</batch_query>
+    <batch_submit>sbatch</batch_submit>
+    <batch_cancel>scancel</batch_cancel>
+    <batch_directive>#SBATCH</batch_directive>
+    <jobid_pattern>(\d+)$</jobid_pattern>
+    <depend_string>--dependency=afterok:jobid</depend_string>
+    <depend_allow_string>--dependency=afterany:jobid</depend_allow_string>
+    <depend_separator>:</depend_separator>
+    <walltime_format>%H:%M:%S</walltime_format>
+    <batch_mail_flag>--mail-user</batch_mail_flag>
+    <batch_mail_type_flag>--mail-type</batch_mail_type_flag>
+    <batch_mail_type>none</batch_mail_type>
+    <submit_args>
+      <arg flag="--time" name="$JOB_WALLCLOCK_TIME"/>
+      <arg flag="-p" name="$JOB_QUEUE"/>
+      <arg flag="--account" name="$PROJECT"/>
```

Comment:
I don't think there's a PROJECT entry for mappy. Will this cause cime to add `--account ` (with no arg) to the submit line? I guess the answer is no, since you tested, but maybe it just happened to work for other reasons, so better double check

Reply:
It looks like it just skips that argument if $PROJECT is empty.

Reply:
Interesting. If that's a reliable feature of the scripts that handle submission, then it's fine.